packages/rum-core/src/domain/rumSessionManager.ts

```diff
@@ -28,17 +28,21 @@ export function startRumSessionManager(configuration: RumConfiguration, lifeCycl
     computeSessionState(configuration, rawTrackingType)
   )
 
-  sessionManager.expireObservable.subscribe(() => {
-    lifeCycle.notify(LifeCycleEventType.SESSION_EXPIRED)
+  sessionManager.expireObservable.subscribe((rawTrackingType: string) => {
+    if (isTypeTracked(rawTrackingType as RumTrackingType)) {
+      lifeCycle.notify(LifeCycleEventType.SESSION_EXPIRED)
+    }
   })
 
-  sessionManager.renewObservable.subscribe(() => {
-    lifeCycle.notify(LifeCycleEventType.SESSION_RENEWED)
+  sessionManager.renewObservable.subscribe((rawTrackingType) => {
+    if (isTypeTracked(rawTrackingType as RumTrackingType)) {
+      lifeCycle.notify(LifeCycleEventType.SESSION_RENEWED)
+    }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏no unit tests require those checks, do we need them?

Reply:
Yes we do :)